go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/external-dns
 
-go 1.14
+go 1.13
```

Comment:
My opinion is to keep this at 1.14. Just make the docs match what is in `go.mod`.